Docs/reference/content/reference/driver/ssl.md

```diff
@@ -43,14 +43,66 @@ var settings = new MongoClientSettings
 
-Prior to v2.7.0, the driver enabled certificate revocation checking by default, in contrast to the `mongo` shell and other MongoDB drivers. This was also in contrast to .NET's defaults for `SslStream` (see .NET Framework documentation [here](https://docs.microsoft.com/en-us/dotnet/api/system.net.security.sslstream.authenticateasclient?view=netframework-4.7.2#System_Net_Security_SslStream_AuthenticateAsClient_System_String_) and .NET Standard documentation [here](https://docs.microsoft.com/en-us/dotnet/api/system.net.security.sslstream.authenticateasclient?view=netstandard-2.0#System_Net_Security_SslStream_AuthenticateAsClient_System_String_)).
+#### Default behavior
+The .NET Driver now **enables** certificate revocation checking by
+default, setting [`CheckCertificateRevocation`]({{< apiref
+"P_MongoDB_Driver_SslSettings_CheckCertificateRevocation">}}) in
+[`SslSettings`]({{< apiref "T_MongoDB_Driver_SslSettings" >}}) to
+`true` by default. This is in contrast to .NET's defaults for
+`SslStream` (see .NET Framework documentation
+[here](https://docs.microsoft.com/en-us/dotnet/api/system.net.security.sslstream.authenticateasclient?view=netframework-4.7.2#System_Net_Security_SslStream_AuthenticateAsClient_System_String_)
+and .NET Standard documentation
+[here](https://docs.microsoft.com/en-us/dotnet/api/system.net.security.sslstream.authenticateasclient?view=netstandard-2.0#System_Net_Security_SslStream_AuthenticateAsClient_System_String_)).
+Any applications relying on the older default of `false` now must
+explicitly set [`CheckCertificateRevocation`]({{< apiref
+"P_MongoDB_Driver_SslSettings_CheckCertificateRevocation">}}) to
+`false` in [`SslSettings`]({{< apiref "T_MongoDB_Driver_SslSettings"
+>}}) to disable certificate revocation checking. Alternatively,
+applications may also set `tlsDisableCertificateRevocationCheck=true`
+in their connection string.  See
+[tlsDisableCertificateRevocationCheck](#tlsDisableCertificateRevocationCheck)
+for more information.
+
+Prior to v2.7.0, the driver also enabled certificate revocation checking by
+default.
+
+#### tlsDisableCertificateRevocationCheck
+The URI option, `tlsDisableCertificateRevocationCheck` controls
+whether or not to disable certificate revocation checking during a TLS
+handshake. Setting `tlsDisableCertificateRevocationCheck=true` is
+equivalent to setting [`CheckCertificateRevocation`]({{< apiref
+"P_MongoDB_Driver_SslSettings_CheckCertificateRevocation">}}) in
+[`SslSettings`]({{< apiref "T_MongoDB_Driver_SslSettings" >}}) to
+`false`.
+
+### OCSP
+
+#### Stapling
+Due to limitations in .NET, the driver currently only supports OCSP
+(Online Certificate Status Protocol) stapling on .NET Core ≥2.x on
+macOS.
+
+On Windows, when a server has Must-Staple certificate and does not
```

Comment:
Do you need to add "a" as in "has a Must-Staple certificate??

Reply:
Oops, thanks for the catch!